citizennet/purescript-ocelot
# FBCM-3677 Social Icon Update

## What does this pull request do?

We have a new batch of icons we need to add support for. We do the work here of updating the icon font, making any necessary updates in the CSS and PS to the existing icons that were updated, as well as the necessary additions to both for the newly added icons.

We also document the process of updating the icon font to make it easier for other engineers to contribute.

## Where should the reviewer start?

Commit-by-commit stops being useful when you get to the documentation updates (the last four commits), by that point I would just look at the final result.

## How should this be manually tested?

- [ ] `yarn run build-all`
- [ ] Load the Icons page of the UI Guide.
  - If developing locally, `open dist/index.html` and navigate to the the "Icons" page.
  - If developing on EC2, navigate to `$(hostname)/ocelot/#icons` (see the `.k8s` folder at the root of this project for instructions on deploying UI Guide, or if that's too much trouble, you can preview the changes here: https://z.aws.citizennet.com/ocelot/#icons).
- [ ] Verify new icon set 
    ![Screen Shot 2020-09-01 at 6 04 45 PM](https://user-images.githubusercontent.com/709141/91920376-b3cf5d00-ec7d-11ea-8080-ed210515f674.png)
